NewHaven/src/Resources.h

```diff
@@ -10,10 +10,10 @@ std::ostream& operator<<(std::ostream& output, const ResourceTypes& resource); /
 struct HarvestTile
 {
 private:
-	ResourceTypes* tileContent;	//Array of resource types
 
-public:
 
+public:
+    ResourceTypes* tileContent;	//Array of resource types
```

Comment:
Tile content was made public for testing purposes, make it private again when the testing is done.